encrypt_credentials_excel.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+import sys
+import yaml
+import getpass
+import argparse
+import os
+from ansible.parsing.vault import VaultEditor, VaultSecret, is_encrypted
+from openpyxl import load_workbook
+
+
+class ExcelParseError(Exception):
+    pass
+
+
+class ExcelParser(object):
+
+    def __init__(self):
+        self.settings = {
+            "column_offset": 1,
+            "row_offset": 4,
+            "row_sections_present": True,
+            "use_list_name": False,
+            "default_fields_by_col": True}
+
+        self.errors = list()
+        self.cell_positions = dict()
+
+    def read_and_encrypt_credentials_excel_sheet(self, passcode, spreadsheet_path):
+        with open('schemas/credentials.json') as credentials_schema:
+            schema_data = yaml.load(credentials_schema.read().decode("utf-8"),
```

Comment:
The 'decode' part here doesn't work in Python 3 because all strings are unicode and already decoded. This is something that still works in Python 2. The Python 3 solution is to just remove the 'decode' part - could this cause issues for users who run this script using Python 2? Or, is this going to be part of the v5 release (will we still be testing Python 2 compatibility in v5?)

Reply:
Use from builtins import str
Just like we use in encrypt_credentials.py